internal/pkg/template/state.go

```diff
@@ -9,17 +9,17 @@ type _state = *state.State
 
 type State struct {
 	_state
-	template *EvaluatedTemplate
+	container *ObjectsContainer
```

Comment:
@JakubMatejka 
Premenoval som to z `EvaluatedTemplate -> template.ObjectsContainer`.
Kedze tu strukturu potrebujem aby implementovala interface `state.ObjectsContainer`.
Nic lepsie mi nenapada.

Co myslis?

Reply:
Jo, třeba. To jsme i mohli nechat jestli není nic trefnějšího. 😃